api.go

```diff
@@ -52,22 +52,22 @@ func writeSuccess(w http.ResponseWriter) {
 	resp := &StatusResponse{Ok: true}
 	out, _ := json.Marshal(resp)
 	w.WriteHeader(http.StatusOK)
-	w.Write(out)
+	_, _ = w.Write(out)
```

Comment:
this is really what should happen for all methods that return values that we don't care about? 😮

Reply:
Well, to ignore error values. We don't really care if we failed to write the response to a status message (probably the caller hung up)